hermes/probe/create/create_file.go

```diff
@@ -0,0 +1,130 @@
+package create
+
+import (
+	"context"
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"math/rand"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/cloudprober/logger"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	m "github.com/googleinterns/step224-2020/hermes/probe/metrics"
+)
+
+const (
+	begin            = 1    // ID of the first HermesFile
+	end              = 50   // ID of the last HermesFile
+	maxFileSizeBytes = 1000 // maximum allowed file size in bytes
+)
+
+type RandomHermesFile struct {
```

Comment:
I'm just call this `GeneratedFile` - you don't need hermes, because it comes from Hermes, and users don't need to know that it's randomly generated.

Reply:
called it RandomFile struct!